tests/virtual_machines_test.go

```diff
@@ -515,27 +515,46 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		})
 
 		Context("When the leader is changed", func() {
-			It("should be able to create a new virtual machine", func() {
+			var (
+				vm *kubevirtv1.VirtualMachine
+			)
+			BeforeEach(func() {
+				By("Restart kubemacpool to reset cache and mac range")
```

Comment:
Can we put this `By` iniside `initKubemacpoolParams` ? if it's not already there.